tests/components/tradfri/test_config_flow.py

```diff
@@ -130,126 +130,6 @@ async def test_discovery_connection(hass, mock_auth, mock_entry_setup):
     }
 
 
-async def test_import_connection(hass, mock_auth, mock_entry_setup):
```

Comment:
I have assumed these tests are related to importing configuration from yaml?

Reply:
yup